src/components/cms/forms/fields/SignatureField.tsx

```
@@ -30,16 +30,18 @@ export function SignatureFieldPreview({ field }: { field: FormFieldBase }) {
-    id: field?.id || '',
     type: FormFieldType.SIGNATURE,
-    label: 'Signature',
+    label: 'Signature Pad',
     name: 'signatureField',
-    helpText: '',
+    helpText: 'Please sign in the box above.',
     isRequired: false,
-    order: field?.order || 0,
     width: 100,
-    options: { backgroundColor: '#ffffff', penColor: '#000000' },
+    options: { canvasHeight: 150, canvasWidth: 300, lineColor: '#000000' },
```

Biome (1.9.4)

[error] This property value named options is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named options


(lint/suspicious/noDuplicateObjectKeys)